client/components/Bomb/Bomb.js

```diff
@@ -939,6 +972,13 @@ class Bomb extends Component {
     this.removeTarget(wire)
   }
 
+  handleLetters = wire => {
+    this.props.passModule('Letters')
+    this.handlePass('module3')
+    this.module1.remove(wire)
```

Comment:
you have this.module1.remove(wire) here, which isn't needed since this function is for module3

Reply:
the removeTarget call down below also needs to remove the letter, not a wire